GiniComponents/GiniInternalPaymentSDK/Sources/GiniInternalPaymentSDK/PaymentInfo/PaymentInfoConfiguration.swift

```diff
@@ -25,6 +25,8 @@ public struct PaymentInfoConfiguration {
     let linksColor: UIColor
     let separatorColor: UIColor
     let backgroundColor: UIColor
+    let closeIcon: UIImage?
+    let closeIconTintColor: UIColor
```

Comment:
[Clarification]: `closeIconTintColor` should it be also optional, since it's needed only if we have an image and it will be will `nil` in Health SDK or?